third_party/tonic/logging/dart_error.h

```diff
@@ -7,13 +7,50 @@
 
 #include "third_party/dart/runtime/include/dart_api.h"
 
+#include "tonic/dart_persistent_value.h"
+
 namespace tonic {
 
 namespace DartError {
+using UnhandledExceptionReporter = void (*)(Dart_Handle, Dart_Handle);
+
 extern const char kInvalidArgument[];
 }  // namespace DartError
 
-bool LogIfError(Dart_Handle handle);
+/// Check if a Dart_Handle is an error or exception.
+///
+/// If it is an error or exception, this method will return true.
+///
+/// If it is an unhandled error or exception, first a call will be made to the
```

Comment:
Is this still accurate?

IIUC the current implementation will call the function given to `SetUnhandledExceptionReporter` for any unhandled exception.

The engine provides a `SetUnhandledExceptionReporter` handler that will try the first available option in this list:
* the platform configuration's `on_error` closure
* the embedder's `Settings.unhandled_exception_callback` function
* a default handler that logs the error

Reply:
Yes, I'll update this.